### DISCUSSION TITLE
Fine Tuning Error - "Cannot specify both model and engine"

### DISCUSSION THREAD

**Discussion by davidgwright** [betalgo/openai#65] [General]
I am using your sample code and am now creating a fine tune test utilising your helper.

I cannot get it to work.

```
            var completionResult = await sdk.Completions.CreateCompletion(new CompletionCreateRequest()
            {
               MaxTokens = 1,
               Prompt = @"https://t.co/f93xEd2 Excited to share my latest blog post! ->",
               Model = _FineTuneResponse.FineTunedModel,
               LogProbs = 2
            });
```

With this code I get an ArgumentException the engineId cannot be null. If I specify an engineId in the line above or set the default on the IOpenAIService I get an error from Open AI stating "Cannot specify both model and engine".

**Comment by kayhantolga** [comment]
I think they changed something with the API. There is another discussion similar to your case. I will have a look as soon as possible.

**Reply by kayhantolga** [reply]
I found what is wrong I will be releasing a fix for model/engine usage this week.

**Reply by davidgwright** [reply]
Is there an Issue number related to this fix? if not, how do I know when fix is released?

**Reply by kayhantolga** [reply]
https://github.com/betalgo/openai/issues/75

**Reply by kayhantolga** [reply]
solved in v6.6.5, please report if you have any issue
